config/prod/tom-rstudio.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
+stack_name: tom-rstudio
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0226a8af83fcecb43"
```

Comment:
[Optional] would be helpful to add a comment telling people where this AMI came from.